SWIG error when using make

### Describe the bug

I get the following error on Windows 10 when running make in python SWIG
`C:\Users\MOHANN~1\AppData\Local\Temp\cciE8lgl.o:cc_mingw.c:(.text+0x5f): undefined reference to `pj_ansi_strxcpy'
collect2.exe: error: ld returned 1 exit status
make: *** [Makefile:43: gcc.exe] Error 1`

### Steps to reproduce

1. Add config_site.h to pjlib/include/pj/ including the config_site_sample.h
2. Build project using Visual Studio 2022
3. Go to pjsip-apps/src/swig/python/ 
4. Run make
5. Get the error

### PJSIP version

2.14.1

### Context

This issue happens on Windows 10, with SWIG version 4.0.0 or SWIG version 4.2.1, GCC version (MinGW.org GCC-6.3.0-1) 6.3.0 ,Python version 3.12.5. 

### Log, call stack, etc

```shell
'sed' is not recognized as an internal or external command,
operable program or batch file.
gcc -o gcc.exe cc_mingw.c
cc_mingw.c: In function 'find_gcc':
cc_mingw.c:44:5: warning: implicit declaration of function 'pj_ansi_strxcpy' [-Wimplicit-function-declaration]
     pj_ansi_strxcpy(spath, p, sizeof(spath));
     ^~~~~~~~~~~~~~~
C:\Users\MOHANN~1\AppData\Local\Temp\cciE8lgl.o:cc_mingw.c:(.text+0x5f): undefined reference to `pj_ansi_strxcpy'
collect2.exe: error: ld returned 1 exit status
make: *** [Makefile:43: gcc.exe] Error 1
```

--- a/pjsip-apps/src/swig/python/cc_mingw.c
+++ b/pjsip-apps/src/swig/python/cc_mingw.c
@@ -41,7 +41,7 @@ const char* find_gcc(const char *gcc_exe)
         return NULL;
     }
 
-    pj_ansi_strxcpy(spath, p, sizeof(spath));
+    strncpy(spath, p, sizeof(spath));
     p = strtok(spath, ";");
     while (p) {
         int len;

